software/spmd/nprimes/Makefile

```diff
@@ -23,23 +7,11 @@ bsg_tiles_Y = 1
 
 all: main.run
 
-# Rule to write processor execution logs. To be used after the
-# verilog simulation.
-#
-# Redirects verilog standard output starting with "X<x_cord>_Y<y_cord>.pelog" 
-# to a unique log file for each coordinate in the manycore. This can be useful 
-# for a quick debug of processor or program running on it.
-proc_exe_logs: X0_Y0.pelog X1_Y0.pelog
 
 OBJECT_FILES=main.o
 
 include ../Makefile.include
 
-#########################################################
-#            FPU OPTION
-#     The number of horizon node must be two and must 
-#     be vanilla core 
-BSG_FPU_OP=0
```

Comment:
Out of curiosity, what was BSG_FPU_OP originally for? When we had two versions of Manycore?

Reply:
I don't know. you can ask Shaolin

Reply:
I'm asking you though. Do you know why it's deprecated and needs to be removed? Or is it just no longer used anywhere and we believe we can safely remove it?

Reply:
I see it in here, which looks like something for design compiler: 
https://github.com/bespoke-silicon-group/bsg_manycore/blob/master/syn/Makefile.dc

Reply:
It's not used as anywhere. that Makefile is also not used.

Reply:
OK, so BSG_FPU_OP is no longer used. If you're going to remove it let's remove it everywhere. 

```software/spmd/bsg_cache_miss/Makefile:BSG_FPU_OP=0
software/spmd/bsg_cuda_lite_runtime/softmax/Makefile:BSG_FPU_OP=0
software/spmd/bsg_cuda_lite_runtime/log_softmax/Makefile:BSG_FPU_OP=0
software/spmd/bsg_cuda_lite_runtime/conv1d/Makefile:BSG_FPU_OP=0
software/spmd/bsg_hetero_nodes/Makefile:BSG_FPU_OP=0
software/spmd/bypass_core/Makefile:#BSG_FPU_OP=1
software/spmd/crc32/Makefile:BSG_FPU_OP=0
software/spmd/depend_stall_mispredict/Makefile:BSG_FPU_OP=0
software/spmd/deprecated/float1/Makefile:BSG_FPU_OP=1
software/spmd/deprecated/float2/Makefile:BSG_FPU_OP=1
software/spmd/deprecated/shared_fam/Makefile:BSG_FPU_OP=1
software/spmd/dram_addr_mapping/Makefile:BSG_FPU_OP=0
software/spmd/fft/Makefile:BSG_FPU_OP=0
software/spmd/fhello_malloc/Makefile:BSG_FPU_OP=0
software/spmd/fib/Makefile:BSG_FPU_OP=0
software/spmd/float_math/Makefile:BSG_FPU_OP=0
software/spmd/mul_div/Makefile:#BSG_FPU_OP=1
software/spmd/nprimes/Makefile:BSG_FPU_OP=0
software/spmd/quicksort/Makefile:BSG_FPU_OP=0
software/spmd/remote_load_crc/Makefile:BSG_FPU_OP=0
software/spmd/specint2000/Makefile.common:BSG_FPU_OP = 0
software/spmd/striped_hello/Makefile:BSG_FPU_OP=0
software/spmd/striped_struct_vector/Makefile:BSG_FPU_OP=0
software/spmd/striped_vector_sum/Makefile:BSG_FPU_OP=0```